ftw/tabbedview/browser/resources/tabbedview.js

```diff
@@ -307,6 +307,12 @@ load_tabbedview = function(callback) {
       }
 
       tabbedview.view_container.trigger('tab-menu-updated');
+    },
+
+    reset_pagesize: function(pageSize) {
```

Comment:
How does these changes relate to this PR?

Reply:
The `dynamicBatchingForm` resets the pagesize.

Reply:
Your commit message `Use classes instead ids for batching-template to prevent duplicated ids` tells nothing about reset_pagesize. How does this relate to get rid of duplicate id's?

Reply:
It's because I had to adjust the selectors and the implementation itself of the old "workaround" implementation.

This is a kind of refactoring which was necessary for the new implementation.